pdk/1.1/modules/ROOT/pages/policies-pdk-upgrade-pdk.adoc

```diff
@@ -0,0 +1,103 @@
+= Upgrading PDK
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+Flex Gateway Policy Development Kit (PDK) is composed of the following separate versioned components:
+
+* Anypoint CLI PDK plugin
+* PDK Rust libraries
+* Anypoint Cargo plugin
+
+When you create a new custom policy project, the PDK Rust libraries and Anypoint Cargo plugin versions are the same as the Anypoint CLI PDK plugin used to create the project.
+
+To upgrade a custom policy project created with an earlier version, you must upgrade your Anypoint CLI PDK plugin to the latest version. Then, you must modify the relevant files in your custom policy directory to upgrade the PDK Rust libraries and Anypoint Cargo plugin.
+
+[[anypoint-cli-pdk-plugin]]
+== Upgrade the Anypoint CLI PDK Plugin
+
+Your Anypoint CLI PDK plugin is used across all your PDK custom policies. When editing a custom policy created with an earlier plugin version, ensure your <<pdk-rust-libraries, PDK Rust libraries>> and <<anypoint-cargo-plugin, Anypoint Cargo plugin>> match your Anypoint CLI PDK plugin version.
+
+To upgrade your Anypoint CLI PDK plugin to the latest version, execute the following command:
+
+[source,ssh]
+----
+anypoint-cli-v4 plugins:install anypoint-cli-pdk-plugin
+----
+
+To verify your current plugin version, execute the following command:
+
+[source,ssh]
+----
+anypoint-cli-v4 plugins
+----
+
+For example, the following PDK plugin version is `1.1.0`:
+
+[source,ssh]
+----
+anypoint-cli-pdk-plugin 1.1.0
+├─ anypoint-cli-command 1.5.2
+│  └─ @oclif/plugin-help 5.1.12
+└─ @oclif/plugin-version 1.0.4
+----
+
+[[pdk-rust-libraries]]
+== Upgrade the PDK Rust Libraries
+
+The PDK library is the Rust Code library used by a specific custom policy. Different custom policy might use different PDK library versions.
+
+To upgrade the PDK RUST code libraries, modify the `pdk` `dependency` and `pdk-test` `dev-dependency` versions in the `cargo.toml` file of the custom policy. The `pdk` and `pdk-test` versions must match and your Anypoint CLI PDK plugin must support the specified version.
+
+In the following example, both dependency versions are `1.0.0`. Modify it to your desired version:
+
+[source,toml]
+----
+[dependencies]
+pdk = { version = "1.0.0", registry = "anypoint" }
+
+[dev-dependencies]
+pdk-test = { version = "1.0.0", registry = "anypoint" }
+----
+
+[[anypoint-cargo-plugin]]
+== Upgrade the Anypoint Cargo Plugin
+
+The Anypoint Cargo plugin extends the functionality of Rust's package manager Cargo for use with the PDK Rust Libraries.
+
+To upgrade your Anypoint Cargo plugin:
+
+. Open the `Makefile` of your custom policy.
```

Comment:
Don't use the element name in this way. Either say "the `Makefile` file" or refer to it generically "the Makefile".

(LMK if this is unclear...)

Reply:
See the Style Ref [guidance](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-UIelements:~:text=Do%20not%20use%20a%20UI%20label%20to%20replace%20normal%20sentence%20syntax%3A)